README.md

```diff
@@ -1 +1,83 @@
-## Money Exchange
+# Money Exchange
+
+Photo gallery is a web app that is developed on react and provides the best free stock photos, royalty-free images & videos shared by creators.
+
+
+![video presentation](https://user-images.githubusercontent.com/51750930/143566841-63fb0571-90bd-4242-9635-5f930b6e1fe5.mp4)
```

Comment:
- [ ] Please record an additional video containing a highlight about a piece of code that you find interesting or are proud of making.

Reply:
Thank you, I will do it.